ui/search/nuxeo-search-results-layout.js

```diff
@@ -0,0 +1,139 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { dom } from '@polymer/polymer/lib/legacy/polymer.dom.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
+import '../nuxeo-layout.js';
+import { I18nBehavior } from '../nuxeo-i18n-behavior.js';
+
+{
+  /**
+  `nuxeo-search-results-layout`
+  @group Nuxeo UI
+  @element nuxeo-search-results-layout
+  */
+  class SearchResultsLayout extends mixinBehaviors([I18nBehavior], Nuxeo.Element) {
+    static get template() {
+      return html`
+        <nuxeo-layout
+          id="results"
+          href="[[_resultsHref(searchName)]]"
+          model="[[_resultsModel(searchName,nxProvider)]]"
+          error="[[i18n('searchResults.layoutNotFound', searchName)]]"
+          on-element-changed="_formChanged"
+        ></nuxeo-layout>
+      `;
+    }
+
+    static get is() {
+      return 'nuxeo-search-results-layout';
+    }
+
+    static get importMeta() {
+      return import.meta;
+    }
+
+    static get properties() {
+      return {
+        /**
+         * The name of the search layout.
+         * */
```

Comment:
We have an extra * here.
```suggestion
         */
```